### DISCUSSION TITLE
Thumbnail resizing without changing aspect ratio

### DISCUSSION THREAD

**Discussion by matusmisiak** [sn4k3/UVtools#276] [Ideas]
First of all, I am very satisfied with this tool. It was exactly what I was looking for. Only one small thing: couldn't the thumbnails be resized by adding padding/empty space, instead of strecthing/squashing it? It isn't really important, but I think it would be a nice improvement.

**Comment by sn4k3** [comment]
It could... Let me see how that work out in the previews...
